test/powershell/Language/Classes/scripting.enums.tests.ps1

```diff
@@ -3,6 +3,11 @@
 #
 
 Describe 'enums' -Tags "CI" {
+    Context 'Enum definition' {
+        It "Enum values definition have space-insensitive syntax" {
+            { enum E0 {e=0} } | Should Not Throw

```

Comment:
We don't use `[scriptblock]::Create` to test basic parsing issues, so I would prefer just using the same style as the rest of the test.

The linked test is a special case to parse the same text multiple times.